PrimaryInstallScript.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+ 	then
+ 	echo "Permission Denied"
```

Comment:
Same comment as the other PR, there is no direct permission denied here, so it's a misleading error message. That's better left for other commands. What can be done here is to say that this script should be ran as root.